docs/news.rst

```diff
@@ -3,6 +3,113 @@
 Release notes
 =============
 
+.. _release-VERSION:
+
+Scrapy VERSION (unreleased)
+---------------------------
+
+Highlights:
+
+-   Replaced ``start_requests()`` (sync) with :meth:`~scrapy.Spider.start`
+    (async) and changed how it is iterated.
+
+Backward-incompatible changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-   The iteration of start requests and items no longer stops once there are
+    requests in the scheduler, and instead runs continuously until all start
+    requests have been scheduled.
```

Comment:
Is it a backwards incompatible change to start_requests, or is it a behavior of the new `async def start` method?

Reply:
At the moment, it is a backward-incompatible change of `start_requests`. And it is also a backward-incompatible change if you don’t use `start_requests` and instead use `start_urls`.

Shall I make `start_requests` behave lazily as before? Is it OK that `start_urls` has this backward-incompatible behavior by default?

Reply:
Fair enough. I think that's fine to make a backwards-incompatible change here, if it's justified, and the behavior is more reasonable in most cases (which we should try to ensure). Having different behaviors could add to confusion.